IS7024Project/Pages/Shared/_Layout.cshtml

```diff
@@ -22,7 +22,7 @@
                             <a class="nav-link text-dark" asp-area="" asp-page="/Index">Home</a>
                         </li>
                         <li class="nav-item">
-                            <a class="nav-link text-dark" asp-area="" asp-page="/Privacy">Privacy</a>
+                            <a class="nav-link text-dark" asp-area="" asp-page="/ParkList">List of Parks</a>
```

Comment:
I don't think you need a privacy page, it doesn't add anything. I used that spot to redirect to the new page I created that lists your parks.

Reply:
I'd make a net-new nav for List of Parks.  Every website should have a privacy (terms and conditions) page, though it does not necessarily need to be on the top nav.